hvcc/generators/c2fmod/c2fmod.py

```diff
@@ -0,0 +1,115 @@
+import jinja2
```

Comment:
This is missing the GPL copyright notice.
You can use the short form like: https://github.com/Wasted-Audio/hvcc/blob/develop/hvcc/types/meta.py#L1-L4